pkg/common/proto/agent.proto

```diff
@@ -5,6 +5,7 @@ option go_package = ".;common";
 
 service AgentService {
     rpc DraftPlayer(GameState) returns (DraftSelection) {}
+    rpc SubmitFantasyActions(GameState) returns (AttemptedFantasyActions) {}
```

Comment:
Do you need to run make gen?  No generated files changed.

Reply:
ya this was just to show the format, will gen now